### ISSUE TITLE
created at date crash in PCMultipartMessage 

### ISSUE THREAD

**Issue by clay-mindoula** [pusher/chatkit-swift#177]
### What?

Offending line per the attached crash report:
```
let createDate = pcMultiPartMessage.createdAtDate
```
May also be of interest:
```
nsexception
{
  "name": "NSInvalidArgumentException",
  "referenced_object": {
    "address": -9223372036854776000,
    "type": "objc_object",
    "ivars": {
      "tagged_payload": 0
    }
  }
}
reason
-[NSIndexPath countByEnumeratingWithState:objects:count:]: unrecognized selector sent to instance 0x8000000000000000
type
nsexception
```

### Environment

* **PusherChatkit version:**1.4.4
* **Xcode version:**10.2.1
* **Swift version:**4.2
* **Platform(s) running PusherChatkit:**iOS

### Demo Project (if you have one; it helps a lot!)

The symbolicated crash report:

[bugsnag_error_stacktrace_NSInvalidArgumentException_event_5cbeccb90037e2bc06b60000.txt](https://github.com/pusher/chatkit-swift/files/3108527/bugsnag_error_stacktrace_NSInvalidArgumentException_event_5cbeccb90037e2bc06b60000.txt)

----

**Comment by SeanLintern** [comment]
I also encountered a similar issue, it has been fixed in 1.7.2 / 1.7.3

**Comment by OliverPearmain-Pusher** [comment]
Since @SeanLintern implied that this issue is fixed and because we haven't heard anything further in some time I am going to close this issue.  Please re-open with more details if this is still an issue on the latest version of the SDK.
